frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -32,14 +32,14 @@ local VirtualKey = InputContainer:new{
 }
 
 function VirtualKey:init()
-    if self.label == "Sym" or self.label == "ABC" then
-        self.callback = function () self.keyboard:setLayout(self.key or self.label) end
+    if self.label == "Sym" or self.label == "ABC" or self.label == "かな" or self.label == "記号" then
```

Comment:
Okay, I made the changes.  It onlt says I changed so many lines in the layouts because I had to change the indentation.